xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -719,19 +710,10 @@ bool CVideoPlayer::OpenFile(const CFileItem& file, const CPlayerOptions &options
   m_item.SetMimeTypeForInternetFile();
 
   m_bAbortRequest = false;
+  m_error = false;
   m_renderManager.PreInit();
 
-  m_openEvent.Reset();
   Create();
-  if (!CGUIDialogBusy::WaitOnEvent(m_openEvent, g_advancedSettings.m_videoBusyDialogDelay_ms, true))
```

Comment:
Please remove m_videoBusyDialogDelay_ms from CAdvancedSettings as it is now unused.

Reply:
Please remove `#include "dialogs/GUIDialogBusy.h" from this source file as you are no longer using `CGUIDialogBusy` here.